core/src/com/softwarearchitecture/math/Rectangle.java

```diff
@@ -0,0 +1,257 @@
+package com.softwarearchitecture.math;
+
+public class Rectangle {
```

Comment:
Are also all of this functionality needed?

Reply:
I made it in order to use our own rectangle instead of using the rectangle in libgdx